examples/objects/objects.out

```diff
@@ -1,6 +1,6 @@
 # To run this sample, navigate to the directory that contains the
 # `.bal` file, and execute the `ballerina run` command.
 $ ballerina run objects.bal
-{age:0, name:"", parent:()}
-{age:0, name:"", parent:()}
-{age:0, name:"", parent:()}
+{name:"", age:0, parent:null}
```

Comment:
This seems to be due to a bug.

Should be 
```suggestion
{name:"", age:0, parent:""}
```

Reply:
Changed.